Calling/BlobStorageHelper.cs

```diff
@@ -84,6 +85,12 @@ public static async Task<BlobStorageHelperInfo> UploadFileAsync(string connectio
                 FileStream uploadFileStream = File.OpenRead(filePath);
                 BlobContentInfo status = await blobClient.UploadAsync(uploadFileStream, true);
                 uploadFileStream.Close();
+
+                if(filePath.EndsWith("mp4"))
```

Comment:
Is mp4 the only format it will generate?

Reply:
At the moment, we are downloading and uploading recording in mp4 format only. There can be other formats which we didn't explored.